src/Service/ExceptionJsonResponseBuilder.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the OpenapiBundle package.
+ *
+ * (c) Niels Nijens <nijens.niels@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Nijens\OpenapiBundle\Service;
+
+use Exception;
+use Nijens\OpenapiBundle\Exception\HttpExceptionInterface;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+/**
+ * Builds a JSON response to an exception for OpenAPI routes.
+ *
+ * @author David Cochrum <dcochrum@sonnysdirect.com>
+ */
+class ExceptionJsonResponseBuilder implements ExceptionJsonResponseBuilderInterface
```

Comment:
this default implementation builds the response **exactly the same** as `JsonResponseExceptionSubscriber` had done previously

Reply:
fixed the name